apps/updatenotification/src/components/UpdateNotification.vue

```diff
@@ -84,6 +84,11 @@
 			</template>
 		</div>
 
+		<div>
+			{{ t('updatenotification', 'You can change the update channel below which also affects the apps management page.') }} <br>
+			{{ t('updatenotification', 'E.g. after switching to the beta channel, beta app updates will be offered to you in the apps management page.') }}
```

Comment:
```suggestion
			{{ t('updatenotification', 'You can change the update channel below which also affects the apps management page. E.g. after switching to the beta channel, beta app updates will be offered to you in the apps management page.') }}
```
Should be once sentence so it's linked for translators

Reply:
> Good point but I will insert the br tag then...

Please don't add html to the translation. It is not needed to force a line break there.

Reply:
addressed